message PB_UTAChampionRecordUserInfoData {
    optional int32 uid = 1;                                      //!< 玩家UID
    optional int32 plat_id = 2;                                  //!< 所属平台ID
    optional int32 server_id = 3;                                //!< 所属服务器ID
    optional int32 level = 4;                                    //!< 等级
    optional int32 fly_flag = 5;                                 //!< 飞升阶段
    optional int32 top_level = 6;                                //!< 巅峰等级
    optional string guild_name = 7;                              //!< 家族名称
    optional string guild_banner = 8;                            //!< 家族族徽
    optional int32 title_id = 9;                                 //!< 称号
    optional int32 facescore_id = 10;                            //!< 颜值系统头衔资源ID                               
    optional int32 prof_base = 11;                                //!< 职业[0,9)
    optional int32 advance_times = 12;                            //!< 进阶次数[0,6]
    optional int32 avatar_type = 13;                              //!< 角色模型[1,6]
    optional int32 color = 14;                                    //!< 颜色[0,4)
    optional int32 weapon_id = 15;                              //!< 武器ID
    optional int32 sub_weapon_id = 16;                          //!< 副武器ID
    optional int32 headshot_id = 17;                            //!< 头像ID 
    optional int32 special_appearance_type = 18;                //!< 特殊形象类型
    optional int32 special_appearance_param = 19;                //!< 特殊形象参数                               
    repeated int32 wear_surface = 20;                           //!< 已装备的的幻化资源id 数组下标为配置表的type
    optional string role_name = 21;                              //!< 玩家名称
    optional int32 smart_mounts_index = 22;                      //!< 当前骑乘的灵骑 -1:未骑乘
    optional int32 is_use_advanced_flag = 23;                    //!< 灵骑信息使用标识 0:未使用进阶形象 1:使用进阶形象
}

message PB_UTAChampionRecordUserTeamInfoData {
    repeated PB_UTAChampionRecordUserInfoData user_info = 1;                //!< 战队玩家信息
}

message PB_UTAChampionRecordData {
    optional int32 change_state = 1;                                                 // 变更状态
    optional int32 rank = 2;                                                         //!< 第X界
    optional bytes team_name = 3;                                                    //!< 站队名称
    optional PB_UTAChampionRecordUserTeamInfoData user_team_info = 4;                //!< 战队玩家信息
}
